A32NX/layout.json

```diff
@@ -200,6 +200,11 @@
             "size": 313408,
             "date": "132402817714110148"
         },
+        {
+            "path": "html_ui/Fonts/debug.log",
+            "size": 3318,
+            "date": "132402817714110148"
+        },
```

Comment:
was this meant to be checked in?

Reply:
I didn't commit that file so the CI should remove that.